vyxal/context.py

```diff
@@ -6,4 +6,15 @@ class Context:
 
     def __init__(self):
         self.context_values = [0]
-        self.inputs = []
+        self.inputs = [[[], 0]]  # [[[inputs], index], [[inputs], index]]
+        self.number_as_range = False
+        self.online = False
+        self.online_output = ""
+        self.printed = False
+        self.range_start = 1  # Where do auto ranges start?
```

Comment:
Hrm, should this be 0?

Reply:
yesn't.

Reply:
[Try it Online!](https://lyxal.pythonanywhere.com?flags=&code=10%C6%9B%3B&inputs=&header=&footer=)